org.civicrm.bootstrapcivihr/scss/components/_logo.scss

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: 'icomoon';
+  src:
+    url('../fonts/icomoon.ttf?tozcjz') format('truetype'),
+    url('../fonts/icomoon.woff?tozcjz') format('woff'),
+    url('../fonts/icomoon.svg?tozcjz#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+#{$outsideNamespaceMarker} {
+  .chr_logo {
+    display: inline-block;
+    font-family: 'icomoon' !important;
```

Comment:
why `!important`? is there a chance that this font would be overwritten by a parent rule?

Reply:
This comes from the css generated by icomoon, they suggest keeping the `!important`